dipy/direction/closest_peak_direction_getter.pxd

```diff
@@ -16,10 +18,20 @@ cdef class BasePmfDirectionGetter(DirectionGetter):
         double pmf_threshold
         double cos_similarity
 
-    cpdef np.ndarray[np.float_t, ndim=2] initial_direction(
+    cpdef cnp.ndarray[cnp.float_t, ndim=2] initial_direction(
         self,
         double[::1] point)
 
+    cpdef tuple generate_streamline(self,
+                                    double[::1] seed,
+                                    double[::1] dir,
+                                    double[::1] voxel_size,
+                                    double step_size,
+                                    StoppingCriterion stopping_criterion,
+                                    cnp.float_t[:, :] streamline,
+                                    StreamlineStatus stream_status,
+                                    double use_fixed_step)
+
```

Comment:
I do not think this is needed. You do not overwrite it and already defined in `DirectionGetter`

Reply:
ok, thx.